format-xls/src/main/java/io/cdap/plugin/format/xls/input/XlsInputFormatConfig.java

```diff
@@ -0,0 +1,211 @@
+  private static final String NAME_SHEET = "sheet";
+  public static final String NAME_SHEET_VALUE = "sheetValue";
+  private static final String NAME_SKIP_HEADER = "skipHeader";
+  private static final String NAME_TERMINATE_IF_EMPTY_ROW = "terminateIfEmptyRow";
+
+  // properties
+  public static final String NAME_SAMPLE_SIZE = "sampleSize";
+
+  public static final String DESC_SKIP_HEADER =
+    "Whether to skip the first line of each file. The default value is false.";
+  public static final String DESC_SHEET = "Select the sheet by name or number. Default is 'Sheet Number'.";
+  public static final String DESC_SHEET_VALUE = "Specifies the value corresponding to 'sheet' input. " +
+    "Can be either sheet name or sheet no; for example: 'Sheet1' or '0' in case user selects 'Sheet Name' or " +
+    "'Sheet Number' as 'sheet' input respectively. Sheet number starts with 0.";
+  public static final String DESC_TERMINATE_ROW = "Specify whether processing needs to be terminated in case an" +
+    " empty row is encountered while processing excel files. Default value is false.";
+  public static final Map<String, PluginPropertyField> XLS_FIELDS;
+
+  static {
+    Map<String, PluginPropertyField> fields = new HashMap<>(FIELDS);
+    fields.put(NAME_SKIP_HEADER,
+            new PluginPropertyField(NAME_SKIP_HEADER, DESC_SKIP_HEADER, "boolean", false, true));
+    // Add fields specific for excel format handling.
+    fields.put(NAME_SHEET, new PluginPropertyField(NAME_SHEET, DESC_SHEET, "string", false, true));
+    fields.put(NAME_SHEET_VALUE, new PluginPropertyField(NAME_SHEET_VALUE, DESC_SHEET_VALUE, "string", false, true));
+    fields.put(NAME_TERMINATE_IF_EMPTY_ROW, new PluginPropertyField(
+                       NAME_TERMINATE_IF_EMPTY_ROW, DESC_TERMINATE_ROW, "boolean", false, true));
+    XLS_FIELDS = Collections.unmodifiableMap(fields);
+  }
+
+  @Macro
+  @Nullable
+  @Name(NAME_SHEET)
+  @Description(DESC_SHEET)
+  private String sheet;
+
+  @Macro
+  @Nullable
+  @Name(NAME_SHEET_VALUE)
+  @Description(DESC_SHEET_VALUE)
+  private String sheetValue;
+
+
+  @Macro
+  @Nullable
+  @Name(NAME_SKIP_HEADER)
+  @Description(DESC_SKIP_HEADER)
+  private Boolean skipHeader;
+
+  @Macro
+  @Nullable
+  @Name(NAME_TERMINATE_IF_EMPTY_ROW)
+  @Description(DESC_TERMINATE_ROW)
+  private Boolean terminateIfEmptyRow;
+
+  public XlsInputFormatConfig() {
+    super();
+  }
+
+  @VisibleForTesting
+  public XlsInputFormatConfig(@Nullable String schema, @Nullable String sheet, @Nullable String sheetValue,
+                              @Nullable Boolean skipHeader, @Nullable Boolean terminateIfEmptyRow) {
+    super();
+    this.schema = schema;
+    this.sheet = sheet;
+    this.sheetValue =  sheetValue;
+    this.skipHeader = skipHeader;
+    this.terminateIfEmptyRow = terminateIfEmptyRow;
+  }
+
+  public long getSampleSize() {
+    return Long.parseLong(getProperties().getProperties().getOrDefault(NAME_SAMPLE_SIZE, "1000"));
```

Comment:
return an int here instead of a long

Reply:
Using `Integer.parseInt` now, config class now returns int